packages/synapse-react-client/src/components/Webhook/CreateWebhookModal.tsx

```diff
@@ -0,0 +1,380 @@
+                  value={invokeEndpoint}
+                  onChange={e => setInvokeEndpoint(e.target.value)}
+                  required
+                  placeholder={'https://'}
+                />
+                {initialData && (
+                  <>
+                    <Button
+                      onClick={() => {
+                        setShowDeleteConfirmation(true)
+                      }}
+                      color="error"
+                      variant={'outlined'}
+                      startIcon={<DeleteTwoToneIcon />}
+                      sx={{ my: 2 }}
+                    >
+                      Delete Webhook
+                    </Button>
+                    <ConfirmationDialog
+                      title={'Confirm Delete Webhook?'}
+                      onCancel={() => setShowDeleteConfirmation(false)}
+                      open={showDeleteConfirmation}
+                      content={
+                        <>
+                          <Typography variant={'body1'}>
+                            Are you sure you want to delete this webhook? This
+                            action cannot be undone.
+                          </Typography>
+                          {deleteError && (
+                            <Alert severity="error" sx={{ mt: 2 }}>
+                              {deleteError.reason}
+                            </Alert>
+                          )}
+                        </>
+                      }
+                      onConfirm={() => {
+                        handleDelete()
+                      }}
+                      confirmButtonProps={{
+                        children: 'Delete',
+                        color: 'error',
+                        disabled: isDeletePending,
+                      }}
+                      cancelButtonProps={{ disabled: isDeletePending }}
+                    />
+                  </>
+                )}
+              </RevalidationFieldsWrapper>
+            </Grid>
+          </Grid>
+
+          {error && (
+            <Alert severity="error" sx={{ my: 2 }}>
+              {showRequestDomainForm && (
+                <>
+                  Synapse only permits configuring webhooks on verified domains,
+                  which are specified in the{' '}
+                  <Link
+                    href={
+                      'https://rest-docs.synapse.org/rest/index.html#org.sagebionetworks.repo.web.controller.WebhookController'
+                    }
+                    target={'_blank'}
+                  >
+                    REST documentation
+                  </Link>
+                  . You have entered a domain that has not been verified.{' '}
+                  <Link href={NEW_ENDPOINT_CONTACT_FORM_URL} target={'_blank'}>
+                    Please fill out this form to request verification for your
+                    webhook domain
+                  </Link>
+                  .
+                </>
+              )}
```

Comment:
Links to: https://sagebionetworks.jira.com/servicedesk/customer/portal/9/group/16/create/85?summary=Request%20to%20add%20new%20domain%20for%20webhook%20verification&description=%3CPlease%20provide%20the%20domain(s)%20you%20wish%20to%20use%20and%20any%20additional%20details%20related%20to%20your%20request.%3E

![image](https://github.com/user-attachments/assets/f8ea5a24-0154-4d7c-acc6-af9898ddea19)